translations/de-DE.json

```diff
@@ -70,51 +70,57 @@
   "components.Autopay.add_button_text": "Zu Autopay hinzufügen",
   "components.Autopay.add_error": "",
   "components.Autopay.add_success": "",
-  "components.Autopay.close_button_text": "Verstanden",
```

Comment:
It looks like lots of translations have been lost here @korhaliv . I would suggest reverting the translation files to how they in `next` and then rerun `yarn extract-messages` to add just the new strings.